Rakefile

```diff
@@ -7,7 +7,7 @@ rescue LoadError
 end
 
 desc 'run both metrics tasks hourly'
-task hourly: ['record:hourly_release_metrics', 'record:data_freshness', 'record:image_latency']
+task hourly: ['record:hourly_release_metrics', 'record:data_freshness', 'record:image_latency', 'record:spot_price']
```

Comment:
since i want this to run hourly, following the trend of lumping it into this hourly group.